.travis.yml

```diff
@@ -6,9 +6,13 @@ rvm:
   - 2.2.4
 matrix:
   include:
-    - env: OPENSTUDIO_VERSION=2.8.0 && OPENSTUDIO_SHA=55665635f0 && RUBYLIB=/usr/local/openstudio-2.8.0/Ruby:/usr/Ruby
+    # For now use local checkout of model articulation to use the DA branch. This should be removed once
+    # model articulation is released with the bricr model generation file.
+    - env: FAVOR_LOCAL_GEMS=1 OPENSTUDIO_VERSION=2.8.1 OPENSTUDIO_SHA=6914d4f590 RUBYLIB=/usr/local/openstudio-2.8.1/Ruby:/usr/Ruby
+    - env: FAVOR_LOCAL_GEMS=1 OPENSTUDIO_VERSION=2.9.0 OPENSTUDIO_SHA=801faa459c RUBYLIB=/usr/local/openstudio-2.9.0/Ruby:/usr/Ruby
```

Comment:
We should pin to this to OpenStudio 2.9.1 and 3.0. No action now, just documenting.